### PR TITLE
Upgrade image to fix vulnerabilities

### PR DESCRIPTION
**Pull request by michael-jong-ds**
bumped versions of apache tika, redis, rabbitmq, and mariadb

### CODE REVIEW

File: charts/datasaur/templates/start/mariadb-sts.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
     spec:
 {{ include "imagePullSecretTemplate" . | nindent 6 }}
       containers:
-        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}mariadb:10.3.8
+        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}mariadb:10.3.38
```
Comment (daniel-jeremia) [inline charts/datasaur/templates/start/mariadb-sts.yaml:39]: is this ok kak @tonoman3g ? concernnya adalah kalo precedent/point72 helm upgrade pake chart baru, dan image nya g ada (belom di push) malah error

Comment (tonoman3g) [reply charts/datasaur/templates/start/mariadb-sts.yaml:39]: wah, iya, kudu push images dulu..

this is not good

🤔 

apa sebenernya yg begini better dr dockerhub aja ya?

**Review by daniel-jeremia — APPROVED** [review]
Code looks good, please test in Azure @michael-jong-ds

File: charts/datasaur/templates/start/corenlp-deployment.yaml [hunk 2]
```diff
@@ -31,8 +31,12 @@ spec:
     spec:
 {{ include "imagePullSecretTemplate" . | nindent 6 }}
       containers:
+{{ if .Values.dockerHub.enabled }}
+        - image: nlpbox/corenlp:2021-04-18
+{{- else -}}
         - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}nlpbox/corenlp:2021-04-18
```
Comment (daniel-jeremia) [inline charts/datasaur/templates/start/corenlp-deployment.yaml:37]: @tonoman3g ini sherlock dependencies perlu di giniin juga gk kak?

Comment (tonoman3g) [reply charts/datasaur/templates/start/corenlp-deployment.yaml:37]: harusya sih sama ya, third party soale

Comment (michael-jong-ds) [reply charts/datasaur/templates/start/corenlp-deployment.yaml:37]: apakah no changes needed kak?

Comment (tonoman3g) [reply charts/datasaur/templates/start/corenlp-deployment.yaml:37]: yup, no changes needed, ini uda bener

File: charts/datasaur/values.yaml [hunk 3]
```diff
@@ -4,6 +4,9 @@ image:
   pullPolicy: "IfNotPresent"
   internal: false
 
+dockerHub:
```
Comment (tonoman3g) [inline charts/datasaur/values.yaml:7]: berarti by default nanti user bakalan use dockerhub ya?

aman kah buat current customer? 🤔

**Comment by tonoman3g** [comment]
> Code looks good, please test in Azure @michael-jong-ds

hasil testing di azurenya apakah aman? @michael-jong-ds

File: charts/datasaur/values.yaml (hunk 3 above)
Comment (michael-jong-ds) [reply charts/datasaur/values.yaml:7]: harusnya aman, tapi kalo dia installnya airgapped kita harus provide image"nya 
cmiiw @daniel-jeremia

**Comment by michael-jong-ds** [comment]
> hasil testing di azurenya apakah aman? @michael-jong-ds

kemarin testing bareng @daniel-jeremia aman

File: charts/datasaur/values.yaml (hunk 3 above)
Comment (tonoman3g) [reply charts/datasaur/values.yaml:7]: yup, uda diprovide di build.sh dan push-images.sh @michael-jong-ds @daniel-jeremia

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**